src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementProductTable.tsx

```diff
@@ -120,6 +126,12 @@ const ReimbursementProductTable: React.FC<ReimbursementProductTableProps> = ({
                                 InputProps={{
                                   startAdornment: <InputAdornment position="start">$</InputAdornment>
                                 }}
+                                onBlur={(e) =>
```

Comment:
what is onBlur

Reply:
it's called when you click out of the input, I tried using this method with onChange, but it was too glitchy, so I did this instead

Reply:
ok if this is whats required

Reply:
oh its kind of like resigning first responder